core/src/main/java/hudson/PluginManagerUIProxy.java

```diff
@@ -0,0 +1,28 @@
+package hudson;
+
+import jenkins.model.Jenkins;
+
+/**
+ * Extension point for selectively overriding parts of the {@link PluginManager} UI/methods
+ * Anything registered with an @extension can override Jelly and define custom views.
+ * It is also possible to add/modify API calls coming via Stapler, but this requires caution.
+ *
+ * @author Sam Van Oort
+ * @since 1.625
+ */
+public abstract class PluginManagerUIProxy implements ExtensionPoint {
+
+    public PluginManager getManager() {
```

Comment:
Convenience method for extensions, since there are many cases they will want to obtain a ref to the plugin manager.